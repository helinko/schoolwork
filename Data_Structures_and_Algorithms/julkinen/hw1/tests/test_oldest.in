A Captain 2356 2370 Gold Kathryn Janeway
X old
A Commander 2355 2370 Red Chakotay
X old
A Lieutenant 2360 2370 Blue B'Elanna Torres
X old
A Lieutenant 2350 2370 Red Tom Paris
X old
Q

