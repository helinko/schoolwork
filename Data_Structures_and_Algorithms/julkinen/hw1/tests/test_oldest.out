> A Captain 2356 2370 Gold Kathryn Janeway
> X old
Kathryn Janeway Captain 2356 2370 Gold
> A Commander 2355 2370 Red Chakotay
> X old
Chakotay Commander 2355 2370 Red
> A Lieutenant 2360 2370 Blue B'Elanna Torres
> X old
Chakotay Commander 2355 2370 Red
> A Lieutenant 2350 2370 Red Tom Paris
> X old
Tom Paris Lieutenant 2350 2370 Red
> Q
